test/testenv/testenv.go

```diff
@@ -71,10 +71,34 @@ const (
 	// IndexerPod Template String for indexer pod
 	IndexerPod = "splunk-%s-idxc-indexer-%d"
 
-	// MonitoringConsoleSts Montioring Console Statefulset Template
```

Comment:
All of these can be combined into a single formatted string like :
`PVCString = "pvc-%s-splunk-%s-%s-%d"`

Leave an example on how to call like `shcPvc = fmt.Sprintf(PVCString, "etc", deployment.GetName(), "shc-search-head", 1)`